scripts/revng

```diff
@@ -420,6 +424,14 @@ If you're just interested in lifting to LLVM IR, simply use `revng lift`.
        "-o", relative(linked)])
   output = linked
 
+  # Emit debug info with the ad-hoc pass.
+  if args.debuginfo:
+    opt_invocation = build_opt_args(["-S",
+                                    "-debug-info",
+                                    relative(output),
+                                    "-o", "/dev/null"])
+    run(opt_invocation)
+
```

Comment:
Is this feature so important that we want to expose it through `revng-translate`?
This seems to me more of a debug tool to use when needed.
Also, the new autoenforcer would have to implement this, which is a bit annoying.

Reply:
I have to admit it is very useful, especially because if used in conjunction with the `--skip` flag, it enables you basically to take any `.ll`, and apply the debug info emission on it, but I get this may not be an universal need, I'll keep this on my local branch. Changed in 7920dab4d3755c4e576b8de736184a8ce4ddac8e